src/components/victory-gauge.js

```diff
@@ -302,75 +321,176 @@ export default class VictoryGauge extends React.Component {
       .sort(null)
       .startAngle(degreesToRadians(props.startAngle))
       .endAngle(degreesToRadians(props.endAngle))
-      .padAngle(degreesToRadians(props.padAngle))
-      .value((datum) => { return datum.y; });
+      .padAngle(degreesToRadians(props.padAngle));
+  }
+  getSegments(props) {
+    const {segments} = props;
+    if (segments && segments.length) {
+      return segments;
+    }
+    // if there are no segments provide set of data as a
+    // segment that spans entire arc. array of 1?
+    return [1];
+  }
+  getGaugeRange(props, segmentLocations) {
+    const radiansToDegrees = (r) => r * (180 / Math.PI);
+    const {domain} = props;
+    return {
+      minimum: {
+        value: domain && domain[0] || segmentLocations[0].data,
+        degrees: radiansToDegrees(segmentLocations[0].startAngle)
+      },
+      maximum: {
+        value: domain && domain[1] || segmentLocations[1].data,
+        degrees: radiansToDegrees(segmentLocations.reverse()[0].endAngle)
```

Comment:
I understand what you mean, and I'm updating to address the first sentence. The second one I somewhat understand, but I'm struggling to visualize what a chart would look like with a domain that is far outside of the scope of the segments that are fed into it. 

Example: `domain={[10, 100]}` `segments={[12, 13, 14, 15]}`

normally, we would rely on our max domain being the sum of the value of the segments, but obviously, that would be far less than the provided domain here, so I can't picture what a chart like that would look like. I can reason that the minimum domain would scale each value up by 10, so if you wished to display a reading on the second segment it would have to be >= 22 and < 35.